editor/debugger/script_editor_debugger.cpp

```diff
@@ -2508,6 +1702,7 @@ ScriptEditorDebugger::ScriptEditorDebugger(EditorNode *p_editor) {
 		visual_profiler->set_name(TTR("Visual Profiler"));
 		tabs->add_child(visual_profiler);
 		visual_profiler->connect_compat("enable_profiling", this, "_visual_profiler_activate");
+		visual_profiler->connect_compat("break_request", this, "_profiler_seeked");
```

Comment:
The rebase seemed to add this line back https://github.com/godotengine/godot/pull/36340/files ([See the diff](https://github.com/godotengine/godot/pull/36340/files#diff-8c78f209914511f5a7bb1028da3ae7b0))

This line causes an error since the signal `break_request` does not exist.